apps/climatemappedafrica/src/lib/data/blockify/tutorial.js

```diff
@@ -0,0 +1,88 @@
+async function tutorial() {
+  // TODO: Move this to Payload CMS
```

Comment:
Do we already have an issue for this @kelvinkipruto

Reply:
Yes @koechkevin #930